pyproject.toml

```diff
@@ -29,7 +29,6 @@ dependencies = [
   "nibabel",
   "opencv-python",
   "pandas",
-  "pandas-schema",
```

Comment:
Awesome! It seems like we're on the path to eliminating `pandas` too, eh? The [remaining usage](https://github.com/search?q=repo%3Aspine-generic%2Fspine-generic%20pandas&type=code) doesn't seem too bad!